### PR TITLE
Update templates with latest ruleset changes

### PR DESCRIPTION
**Pull request by peppy**

### CODE REVIEW

File: templates/ruleset-empty/osu.Game.Rulesets.EmptyFreeform/Beatmaps/EmptyFreeformBeatmapConverter.cs [hunk 1]
```diff
@@ -13,15 +12,12 @@ namespace osu.Game.Rulesets.EmptyFreeform.Beatmaps
...
     {
-        protected override IEnumerable<Type> ValidConversionTypes { get; } = new[]
-        {
-            // todo: Populate with conversion types that should be supported other than position (ie. typeof(IHasCurve))
-            // https://github.com/ppy/osu/tree/master/osu.Game/Rulesets/Objects/Types
-            typeof(IHasPosition)
-        };
+        // todo: Check for conversion types that should be supported (ie. Beatmap.HitObjects.Any(h => h is IHasXPosition))
+        // https://github.com/ppy/osu/tree/master/osu.Game/Rulesets/Objects/Types
+        public override bool CanConvert() => true;
```
Comment (frenzibyte) [inline templates/ruleset-empty/osu.Game.Rulesets.EmptyFreeform/Beatmaps/EmptyFreeformBeatmapConverter.cs]: methods should reside after the constructor.

File: templates/ruleset-example/osu.Game.Rulesets.Pippidon/Beatmaps/PippidonBeatmapConverter.cs [hunk 2]
```diff
@@ -13,10 +13,10 @@ namespace osu.Game.Rulesets.Pippidon.Beatmaps
 {
     public class PippidonBeatmapConverter : BeatmapConverter<PippidonHitObject>
     {
-        protected override IEnumerable<Type> ValidConversionTypes { get; } = new[] { typeof(IHasPosition) };
+        public override bool CanConvert() => Beatmap.HitObjects.All(h => h is IHasPosition);
```
Comment (frenzibyte) [inline templates/ruleset-example/osu.Game.Rulesets.Pippidon/Beatmaps/PippidonBeatmapConverter.cs]: ditto

File: templates/ruleset-scrolling-empty/osu.Game.Rulesets.EmptyScrolling/Beatmaps/EmptyScrollingBeatmapConverter.cs [hunk 3]
```diff
@@ -11,14 +10,12 @@ namespace osu.Game.Rulesets.EmptyScrolling.Beatmaps
...
     public class EmptyScrollingBeatmapConverter : BeatmapConverter<EmptyScrollingHitObject>
     {
-        protected override IEnumerable<Type> ValidConversionTypes { get; } = new Type[]
-        {
-            // todo: Populate with conversion types that should be supported (ie. typeof(IHasXPosition))
-            // https://github.com/ppy/osu/tree/master/osu.Game/Rulesets/Objects/Types
-        };
+        // todo: Check for conversion types that should be supported (ie. Beatmap.HitObjects.Any(h => h is IHasXPosition))
+        // https://github.com/ppy/osu/tree/master/osu.Game/Rulesets/Objects/Types
+        public override bool CanConvert() => true;
```
Comment (frenzibyte) [inline templates/ruleset-scrolling-empty/osu.Game.Rulesets.EmptyScrolling/Beatmaps/EmptyScrollingBeatmapConverter.cs]: ditto

**Review by frenzibyte — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
Projects have been brought up-to-date with latest osu-side changes.

**Review by frenzibyte — APPROVED** [review]
Seems fine code-quality side.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**